hack/verify-generated-swagger-docs.sh

```diff
@@ -70,7 +70,7 @@ ret=0
 pushd "${KUBE_ROOT}" > /dev/null 2>&1
   # Test for diffs
   _output=""
-  for file in ${TARGET_FILES[*]}; do
+  for file in "${TARGET_FILES[*]}"; do
```

Comment:
TARGET_FILES are computed a few lines above by appending files read one by one into an array.

quoting to prevent whitespace issues.